src/site/content/en/blog/sharedarraybuffer-updates-january-2021/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Firefox and Chrome now support SharedArrayBuffer (again) for cross-origin isolated pages
+subhead: >
+  The API for sharing memory between the main thread and worker threads is available (again)
+  on the desktop and Android versions of Firefox and Chrome. SharedArrayBuffer is only available for
+  websites that have implemented COOP and COEP.
+description: >
+  The API for sharing memory between the main thread and worker threads is available (again)
+  on the desktop and Android versions of Firefox and Chrome. SharedArrayBuffer is only available for
+  websites that have implemented COOP and COEP.
+authors:
+  - kaycebasques
+date: 2021-01-13
+# updated: 2020-07-22
+hero: hero.jpg
+alt: An illustration of a person browsing a website that has a popup, an iframe, and an image.
+tags:
+  - blog
+  - security
+  - performance
+  - memory
+feedback:
+  - api
+---
+
+Here's some quick browser compatibility updates regarding [`SharedArrayBuffer`][mdn]:
+
+* Chrome recently [announced](TODO) that it's enabling `SharedArrayBuffer` in
```

Comment:
If I was a reader, I'd overlook this link. Perhaps link again when we emphasize that this change will break existing implementations?